sdk/eventhub/azure-eventhubs-checkpointstoreblob/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include *.md
+include azure/__init__.py
+include azure/eventhub/__init__.py
+include azure/eventhub/extensions/__init__.py
```

Comment:
must include samples and tests (look for example in azure-core file)

Reply:
updated